### PR TITLE
add swagger docs

### PR DESCRIPTION
**Pull request by Ayudh**

### CODE REVIEW

**Review by abruzgin — COMMENTED** [review]
Great, but where's the main YAML file for Swagger docs?

**Comment by Ayudh** [comment]
@AlexBryuzgin I missed that part. Will update at the EOD. Please consider.

File: javascript-client-generated/swagger.yml [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+          name: body
+          description: Created user object
+          required: true
+          schema:
+            $ref: '#/definitions/User'
+      responses:
+        default:
+          description: successful operation
+securityDefinitions:
+  petstore_auth:
```
Comment (abruzgin) [inline javascript-client-generated/swagger.yml:210]: Seems like some odd code, isn't it?

**Review by abruzgin — APPROVED** [review]

File: javascript-client-generated/swagger.yml (hunk 1 above)
Comment (Ayudh) [reply javascript-client-generated/swagger.yml:210]: Yes. I will remove this.

**Pull request closed without merging**